scripts/qa_crowbarsetup.sh

```diff
@@ -2365,6 +2406,20 @@ function custom_configuration()
             if [[ $hacloud = 1 ]] ; then
                 proposal_set_value manila default "['deployment']['manila']['elements']['manila-server']" "['cluster:$clusternameservices']"
             fi
+
+            if iscloudver 7plus || (iscloudver 6 && ! [[ $cloudsource =~ ^M[1-8]$ ]]); then
+                # new generic driver options since M9
+                if crowbar manila proposal show default|grep service_instance_name_or_id ; then
+                    proposal_set_value manila default "['attributes']['manila']['shares'][0]['backend_driver']" "'generic'"
+                    proposal_set_value manila default "['attributes']['manila']['shares'][0]['backend_name']" "'backend1'"
+                    proposal_set_value manila default "['attributes']['manila']['shares'][0]['generic']['service_instance_user']" "'root'"
+                    proposal_set_value manila default "['attributes']['manila']['shares'][0]['generic']['service_instance_password']" "'linux'"
+                    proposal_set_value manila default "['attributes']['manila']['shares'][0]['generic']['share_volume_fstype']" "'ext3'"
+                    proposal_set_value manila default "['attributes']['manila']['shares'][0]['generic']['service_instance_name_or_id']" "'$manila_service_vm_uuid'"
+                    proposal_set_value manila default "['attributes']['manila']['shares'][0]['generic']['service_net_name_or_ip']" "'$manila_service_vm_ip'"
+                    proposal_set_value manila default "['attributes']['manila']['shares'][0]['generic']['tenant_net_name_or_ip']" "'fixed'"
```

Comment:
The batch file needs to include those settings then .. (it will be difficult to sneak in the uuid though).